src/leafdoc.js

```diff
@@ -114,6 +114,13 @@ Leafdoc.prototype.registerDocumentable = function(name, label) {
 	return this;
 };
 
+// 🍂method getTemplateEngine(): Handlebars
+// Returns handlebars template engine used to render templates.
+// You can use it for override helpers or register new.
+Leafdoc.prototype.getTemplateEngine = function() {
```

Comment:
Aaaah, I got it. That's a nifty idea!

I think we'll need to add some proper documentation about how the templates work, but that can be done later.